ncache-vm-cluster/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
if this is not supported in Azure US Government, add the environments property to select clouds that are supported